dev/tests/integration/framework/Magento/TestFramework/TestCase/AbstractController.php

```diff
@@ -15,6 +15,7 @@
 
 /**
  * @SuppressWarnings(PHPMD.NumberOfChildren)
+ * @SuppressWarnings(PHPMD.CouplingBetweenObjects)
```

Comment:
I've added some changes, which reduced coupling, in a way, which seems reasonable to me. Going to proceed with the merge now. Please let me know if it's ok from your perspective.